PollBuddy-Server/backend/modules/modelUtils.js

```diff
@@ -5,9 +5,10 @@ const getID = function (ID) {
   return new bson.ObjectID(ID);
 };
 
-const getGroupInternal = async function(groupID) {
+const getGroupInternal = async function(groupID, constraints={}) {
   let idCode = new bson.ObjectID(groupID);
```

Comment:
This function is becoming just a replacement to mongoConnect.getDB().... and should be treated that way. Lets remove the groupID parameter and have callee's provide the _id in the parameters. This way we can remove these two lines. Also the idCode line is redundant as groupID is already an object id through the validator.